extensions/common/api/api-core/src/main/java/org/eclipse/dataspaceconnector/api/model/IdResponseDto.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (c) 2022 T-Systems International GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       T-Systems International GmbH
+ *
+ */
+
+package org.eclipse.dataspaceconnector.api.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+@JsonDeserialize(builder = String.class)
```

Comment:
that seems incorrect. should be `IdResponseDto.Builder`. Also, there is no test for this class. There should at least be a test to verify correct serialization and deserialization, which would have surfaced the incorrect builder.